workers/data_refinery_workers/processors/smasher.py

```diff
@@ -117,20 +119,45 @@ def _prepare_files(job_context: Dict) -> Dict:
     log_state("end prepare files", job_context["job"], start_prepare_files)
     return job_context
 
+def downlad_computed_file(download_tuple: Tuple[str, ComputedFile]):
+    """ this function downloads the latest computed file. Receives a tuple as its first parameter
+    where the first item is the path where the computed file needs to be downloaded, and the second
+    parameter is the computed file.
+    This is used to parallelize downloading quantsf files. """
+    (output_file_path, latest_computed_file) = download_tuple
+    try:
+        latest_computed_file.get_synced_file_path(path=output_file_path)
+    except:
+        # Let's not fail if there's an error syncing one of the quant.sf files
+        logger.exception('Failed to sync computed file', computed_file_id=latest_computed_file.pk)
 
 def sync_quant_files(output_path, files_sample_tuple, job_context: Dict):
     """ Takes a list of ComputedFiles and copies the ones that are quant files to the provided directory.
         Returns the total number of samples that were included """
     num_samples = 0
-    for (_, sample) in files_sample_tuple:
-        latest_computed_file = sample.get_most_recent_quant_sf_file()
-        # we just want to output the quant.sf files
-        if not latest_computed_file: continue
-        accession_code = sample.accession_code
-        # copy file to the output path
-        output_file_path = output_path + accession_code + "_quant.sf"
-        num_samples += 1
-        latest_computed_file.get_synced_file_path(path=output_file_path)
+    samples = [sample for (_, sample) in files_sample_tuple]
```

Comment:
Any reason not to use `job_context['samples']` at this point? I'd say that you could just modify `smasher._prepare_files()` to call the appropriate `get_most_recent_*` but function but I'm pretty sure that with https://github.com/AlexsLemonade/refinebio/issues/1751 that you're planning on not calling that at all, which means the key that you're referencing here won't exist at that point anyway.